doc/conf.py

```diff
@@ -55,7 +55,10 @@ def _check_deps():
         try:
             __import__(name)
         except ImportError:
-            missing.append(name)
+            if name == 'PIL.Image':
```

Comment:
just make names a mapping?  IPython.blah.blah is likewise technically not the correct name.